IBM-86: Out of Commission Reasons

cksanders opened the pull request.

chrisns commented:
i'm not sure how I feel about using the +/- pngs as apposed to using font aweome style:
http://fontawesome.io/icon/plus-circle/
or an icon from: https://github.com/alphagov/govuk_frontend_toolkit
I know the govuk standards don't have a plus/minus in there, so perhaps we should make a PR and then npm install the govuk_frontend_toolkit


apart from that, this looks ok but we can't merge this until the backend work is also merged

cksanders commented:
Great idea Chris. Temporarily I have implemented the http://fontawesome.io ; additionally I have created https://jira.digital.homeoffice.gov.uk/browse/IBM-238 in relation to the govuk_frontend_toolkit contribution and added it to the product backlog, pick it up once priority items are done.

chrisns commented:
...re-adding the do not merge label, until the BE and e2e is ready to merge then the three can be all merged together

chrisns commented on app/views/directives/centreGender.html:
```diff
@@ -24,10 +24,15 @@
-      <td ng-bind="ooc"></td>
+      <td ng-bind="ooctotal"></td>
+    </tr>
+    <tr class="ooc" ng-if="showooc" ng-repeat="(key, value) in oocdetail">
```
please include a `track by` in the ng-repeat
